### PR TITLE
[RFR] Fix build for postgres-queries-builder

### PR DESCRIPTION
**Pull request by alexisjanvier**

### CODE REVIEW

File: packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts [hunk 1]
```diff
@@ -23,10 +23,9 @@ export const batchRemove = ({
     const selector = [].concat(primaryKey);
     const idSanitizer = sanitizeIdentifier(selector);
 
-    return ids => {
-        const cleanIds = ids.map(id => idSanitizer(id));
+    return (ids: Array<string | number | StringMap>) => {
```
Comment (jdemangeon) [inline packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts]: This typing is already declared in QueryFunction

File: packages/postgres-queries-builder/src/queries/select/SelectByOrderedIdentifiers.ts [hunk 2]
```diff
@@ -33,7 +33,9 @@ ON ${table}.${primaryKey}::varchar=x.${primaryKey}
 ORDER BY x.ordering;`;
 
     const parameters = flatten(
-        sanitizeParameter([primaryKey], { [primaryKey]: values }),
+        sanitizeParameter([primaryKey], {
+            [Array.isArray(primaryKey) ? primaryKey[0] : primaryKey]: values,
```
Comment (jdemangeon) [inline packages/postgres-queries-builder/src/queries/select/SelectByOrderedIdentifiers.ts]: Are you sure that selecting first value of the array is the expected way of working ?

File: packages/postgres-queries-builder/src/queries/select/SelectOne.ts [hunk 3]
```diff
@@ -27,7 +27,11 @@ export const selectOne = ({
         const parameters = sanitizeIdentifier(identifiers, {
             ...(typeof rawParameters === 'object'
                 ? rawParameters
-                : { [primaryKey]: rawParameters }),
+                : {
+                      [Array.isArray(primaryKey)
```
Comment (jdemangeon) [inline packages/postgres-queries-builder/src/queries/select/SelectOne.ts]: Same

File: packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts (hunk 1 above)
Comment (alexisjanvier) [reply packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts]: Yes, I know, but it wasn't enough ....  Transpilation failed without that.

File: packages/postgres-queries-builder/src/queries/select/SelectByOrderedIdentifiers.ts (hunk 2 above)
Comment (alexisjanvier) [reply packages/postgres-queries-builder/src/queries/select/SelectByOrderedIdentifiers.ts]: Nope ... But: 
* tests don't fail,
* giving an array as an index seems like a bad idea...

File: packages/postgres-queries-builder/src/queries/select/SelectOne.ts (hunk 3 above)
Comment (alexisjanvier) [reply packages/postgres-queries-builder/src/queries/select/SelectOne.ts]: Same response

File: packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts (hunk 1 above)
Comment (jdemangeon) [reply packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts]: Have you tried to put Array<string | number | StringMap> typing in QueryFunction def ?

File: packages/postgres-queries-builder/src/queries/select/SelectByOrderedIdentifiers.ts (hunk 2 above)
Comment (jdemangeon) [reply packages/postgres-queries-builder/src/queries/select/SelectByOrderedIdentifiers.ts]: :+1:

File: packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts (hunk 1 above)
Comment (alexisjanvier) [reply packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts]: Hey, it's working ! Les voies du TypeScript sont encore impénétrables pour moi :(

**Pull request merged**